docs/user-guide.rst

```diff
@@ -82,8 +82,8 @@ Response Content
 ----------------
 
 The :class:`~response.HTTPResponse` object provides
-:attr:`~response.HTTPResponse.status`, :attr:`~response.HTTPResponse.data`, and
-:attr:`~response.HTTPResponse.headers` attributes:
+:attr:`~response.BaseHTTPResponse.status`, :attr:`~response.HTTPResponse.data`, and
+:attr:`~response.BaseHTTPResponse.headers` attributes:
```

Comment:
This case is different from `request()`. Unlike `RequestMethods` we may want to show the inheritance explicitly in the documentation (i.e. document `headers` in `BaseHTTPResponse`, and `data` in `HTTPResponse`), so this may be correct. (Note: this change not lower the number of warnings because that attribute is not documented and `BaseHTTPResponse` isn't currently part of the reference. Here's how you can [document attributes](https://github.com/python-trio/trio/blob/4b0388b7c9442e9bea5259460cd64687bff8c26b/trio/_subprocess.py#L86-L107))

@sethmlarson Do we want to document `BaseHTTPResponse` separately or should each subclass documentation contain the inherited members docs?

Reply:
Dicussed a bit offline with Seth, and he confirmed that we don't want `BaseHTTPResponse` to be in the actual docs.